File: projetoConceitos/app/src/main/java/br/edu/qi/projetoconceitos/MainActivity.java
package br.edu.qi.projetoconceitos;

import androidx.appcompat.app.AppCompatActivity;

import android.os.Bundle;
import android.view.View;
import android.widget.Button;
import android.widget.EditText;
import android.widget.TextView;
import android.widget.Toast;

public class MainActivity extends AppCompatActivity {
    EditText editTextNome;
    EditText editTextMedia;
    Button buttonVerificar;
    TextView TextViewResultado;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);
        editTextNome = findViewById(R.id.editTextNome);
        editTextMedia= findViewById(R.id.editTextMedia);
        buttonVerificar = findViewById(R.id.buttonVerificar);
        TextViewResultado = findViewById(R.id.textViewResultado);
    }

    public void mostrarDadosDoAluno(View view) {
        Aluno objetoAluno = new Aluno();
        if (editTextNome.getText().toString().isEmpty() || editTextMedia.getText().toString().isEmpty()){
            Toast.makeText(this, "Complete os campos", Toast.LENGTH_LONG).show();
        }else {
            objetoAluno.setNome(editTextNome.getText().toString());
            objetoAluno.setMedia(Double.parseDouble(editTextMedia.getText().toString()));
            TextViewResultado.setText(objetoAluno.toString());
        }
    }

}

File: ProvaRefeitaEstruturada/app/src/main/java/junior/henrique/provarefeitaestruturada/MainActivity.kt
package junior.henrique.provarefeitaestruturada

import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import android.widget.Button
import kotlinx.android.synthetic.main.activity_main.*
import org.jetbrains.anko.AnkoAsyncContext
import org.jetbrains.anko.AnkoLogger
import org.jetbrains.anko.startActivity

class MainActivity : AppCompatActivity(), AnkoLogger {

    lateinit var button_prova: Button

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)

        initialize()

        button_prova.setOnClickListener {
            startActivity<HomeActivity>()
        }
    }

    fun initialize() {
        button_prova = findViewById(R.id.button_prova)
    }
}
